buildconfig/CMake/FindSphinx.cmake

```diff
@@ -12,7 +12,7 @@
 #=============================================================
 
 find_program( SPHINX_EXECUTABLE NAME sphinx-build.cmd sphinx-build
-  PATHS ${CMAKE_LIBRARY_PATH}/python2.7/Scripts
+  PATHS ${CMAKE_LIBRARY_PATH}/python${PYTHON_MAJOR_VERSION}.${PYTHON_MINOR_VERSION}/Scripts
```

Comment:
Oh joy.

I think the PATHS are in addition to the standard system paths as that shouldn't work for Ubuntu now. Hopefully we can just detect whether we are using `3` and then have a list of known names for that. As you say, I'll leave that for another day.